packages/combo-box/test/external-filtering.test.js

```diff
@@ -65,17 +57,6 @@ describe('external filtering', () => {
       expect(getFocusedItemIndex(comboBox)).to.equal(1);
     });
 
-    it('should reset focus when opening with filter cleared', () => {
```

Comment:
**note:** This test has been moved to:

https://github.com/vaadin/web-components/pull/3600/files#diff-95666241933a0971a1c2958cbd8143ccfbd4b269d293ab5b42180fe539c985b7R122-R125